src/platforms/android/common/dependencies.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Managing Dependencies
+sidebar_order: 1000
+---
+
+## Managing versions for multiple Sentry dependencies
+
+When multiple Sentry dependencies are used, to avoid specifying the version of each dependency, you can use a *BOM* or *Bill Of Materials*.
+
+Using Maven you can add the following to `dependencyManagement` section in your `pom.xml`:
+
+```xml {tabTitle:Maven}{filename:pom.xml}
```

Comment:
Does it make sense to add the maven information on Android? Wouldn't Android be 90% (or more?) gradle?

Reply:
if its an Android page only, only Gradle would be fine

Reply:
👍